lang/css/css.talon

```diff
@@ -0,0 +1,46 @@
+tag: user.css
+-
+tag(): user.code_comment_block_c_like
+tag(): user.code_functions_common
+tag(): user.code_libraries
+tag(): user.code_operators_math
+
+settings():
+    user.code_public_variable_formatter = "DASH_SEPARATED"
+
+block: user.code_block()
+
+attribute [<user.text>]:
+    name = user.formatted_text(text or "", "DASH_SEPARATED")
+    user.insert_between("[{name}", "]")
+
+prop <user.text>:
+    name = user.formatted_text(text, "DASH_SEPARATED")
+    user.insert_between("{name}: ", ";")
+
+# for media/supports queries, or if you don't like prop
+rule <user.text>:
+    name = user.formatted_text(text, "DASH_SEPARATED")
+    insert("{name}: ")
+
+value <user.number_string> [{user.css_unit}]:
+    "{number_string}{css_unit or ''}"
+value <user.number_string> point <digit_string> [{user.css_unit}]:
+    "{number_string}.{digit_string}{css_unit or ''}"
+
+(value|state) {user.css_global_value}: "{css_global_value}"
+value <user.text>: user.insert_formatted(text, "DASH_SEPARATED")
+
+variable <user.text>:
+    name = user.formatted_text(text, settings.get("user.code_public_variable_formatter"))
+    insert("var(--{name})")
+
+op var: user.insert_between("var(--", ")")
+
+at {user.css_at_rule}: "@{css_at_rule} "
+unit {user.css_unit}: insert(css_unit)
+
+[value] current color: "currentColor"
+important: " !important"
+
+border box model: "box-sizing: border-box;\n"
```

Comment:
border box model and value current color might be a bit idiosyncratic to your case, I wouldn't have used them that often the way I've worked with CSS.

Reply:
border box model might be slightly idiosyncratic - it probably most commonly occur in CSS resets these days - notably Bootstrap makes that the default. I'm fine removing it for knausj.

I suppose for `currentColor` people might want to spell it `currentcolor` all lowercase? I'd argue it's pretty useful to set *-color properties without hardcoding them. Happy to remove that on the preference note though.

Reply:
Happy to leave in the currentColor thing if you reckon it's useful. I'll drop out the border box model command and then merge this in, thanks!